### DISCUSSION TITLE
ERROR: Retrieval of PROTO 'Create.proto' was unsuccessful, the asset should be cached but it is not.

### DISCUSSION THREAD

**Discussion by flyorfight** [cyberbotics/webots#6541] [General]
what's wrong with this error?I have downloaded the "projects" document from the "webots-r2022b.zip"?But when I wanted to accelerate a robot ,the error come out!

**Comment by ShuffleWire** [comment]
Please describe more precisely the step you took and your setup (OS, webots version...)

Moreover, is there any reason to use webots 2022 ? there are many improvement about the asset management that came with webots 2023.

And what do you mean by "accelerate a robot" ?

This file should be in `webots/projects/robots/irobot/create/protos/` do you have it there ?
